nais/nais.yaml

```diff
@@ -28,6 +28,12 @@ spec:
     outbound:
       external:
         - host: {{rekrutteringsbistand-api-host}}
+  elastic:
+    instance: rekrutteringsbistand-stilling
+  envFrom:
+    - secret: rekrutteringsbistand-stilling-indekser-es-credentials
```

Comment:
Denne kommer fra _Kubernetes secrets_. Se alle secrets ved med `kubectl get secrets` når du er i namespace arbeidsgiver i context dev-gcp. Se denne secreten (med Base64-encodede verdier) med:

```
kubectl get rekrutteringsbistand-stilling-indekser-es-credentials -o yaml
```

Reply:
Secreten er opprettet manuelt med
```
kubectl create secret generic rekrutteringsbistand-stilling-indekser-es-credentials --from-literal=ES_USERNAME=<bruker> --from-literal=ES_PASSWORD=<passord>
``` 

Brukernavn og passord har jeg fått fra Kyrre, som hentet det fra Aiven-konsollen. Den gir både read- og write-rettigheter. Vi har også brukere for kun read, write og en admin-bruker.